pkg/action/mock_action.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// mockgen -source pkg/action/action.go -destination pkg/action/mock_action.go
+// Code generated by MockGen. DO NOT EDIT.
+// Source: pkg/action/action.go
+
+// Package action is a generated GoMock package.
+package action
+
+import (
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+	meta "k8s.io/apimachinery/pkg/api/meta"
+	discovery "k8s.io/client-go/discovery"
+	rest "k8s.io/client-go/rest"
+)
+
+// MockRESTClientGetter is a mock of RESTClientGetter interface
+type MockRESTClientGetter struct {
```

Comment:
Can you move this to pkg `internal/test`? It should not be a part of the public API.

Reply:
okay 👍

Reply:
I've moved the files now 👍